README.md

```diff
@@ -30,19 +30,42 @@ server.plugin.register({
           }
          ]
       }],
-      mongo: require("mongodb")
+      mongo: require("mongodb"),
+      connectionOptions: {
+          server: {
+              poolSize: 15,
+              socketTimeout: 10000
+          },
+          replSet: {
+              connectWithNoPrimary: false
+          }
+      }
   }
 });
 
 ```
 
+`connectionOptions` parameter can be passed either as a child of options or per db. `connectionOptions` set for specific db take precedance over general ones.
```

Comment:
Ok, I see what you are doing it here, you can provide both global settings and connection specific settings. It seems to be a bit unnecessarily complicated and I would probably go for the easier approach to provide the config for each connection.